js/src/main/scala/crystal/react/hooks/UseSerialStateView.scala

```diff
@@ -1,17 +1,15 @@
 package crystal.react.hooks
 
 import crystal.react.View
-import japgolly.scalajs.react.Reusable
+import crystal.react.reuse._
 import japgolly.scalajs.react._
 import japgolly.scalajs.react.hooks.CustomHook
 
 object UseSerialStateView {
-  def hook[A]: CustomHook[A, Reusable[View[A]]] = CustomHook[A]
+  def hook[A]: CustomHook[A, Reuse[View[A]]] = CustomHook[A]
```

Comment:
Is it so that `Reusable` can be easily converted to `Reuse`?

Reply:
This is to unify the API with the new `useStateViewWithReuse` which returns a `Reuse`.

There's an implicit conversion from `Reuse` to `Reusable` if need be.